### PR TITLE
TextEditor: fix some wording

### PR DESCRIPTION
**Pull request by poire-z**
Fix review comments around https://github.com/koreader/koreader/pull/4135#issuecomment-410823170

### CODE REVIEW

File: plugins/texteditor.koplugin/main.lua [hunk 1]
```diff
@@ -381,7 +381,7 @@ function TextEditor:editFile(file_path, readonly)
         buttons = is_lua and {{
             -- First button on first row, that will be filled with Reset|Save|Close
             {
-                text = _("Check lua"),
+                text = _("Check Lua"),
```
Comment (Frenzie) [inline plugins/texteditor.koplugin/main.lua]: Note that there was also lowercase a few lines down

File: plugins/texteditor.koplugin/main.lua [hunk 2]
```diff
@@ -211,7 +211,7 @@ Do you want to proceeed?]]),
                 onConfirm = function(dir_path)
                     local file_input
                     file_input = InputDialog:new{
-                        title =  _("Enter new file filename"),
+                        title =  _("Enter filename"),
```
Comment (Frenzie) [inline plugins/texteditor.koplugin/main.lua]: New filename would also work if it's for a new file, but new file filename is stylistically awkward ;)

Comment (poire-z) [reply plugins/texteditor.koplugin/main.lua]: Yes, new file filename sounded bad to me too :) but Enter filename sounds prettier to my ears now than Enter new filename.

Comment (Frenzie) [reply plugins/texteditor.koplugin/main.lua]: Thunar, for reference:
screenshot_2018-08-07_20-50-40

**Pull request merged**